crucible-debug/src/Lang/Crucible/Debug/Personality.hs

```diff
@@ -0,0 +1,72 @@
+{-|
+Copyright        : (c) Galois, Inc. 2025
+Maintainer       : Langston Barrett <langston@galois.com>
+-}
+
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE FunctionalDependencies #-}
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+
+module Lang.Crucible.Debug.Personality
+  ( HasContext(..)
+  , prepend
+  , quit
+  , stop
+  ) where
+
+import Control.Lens qualified as Lens
+import Data.Function ((&))
+import Lang.Crucible.Debug.Context (Context)
+import Lang.Crucible.Debug.Context qualified as Ctxt
+import Lang.Crucible.Debug.Inputs qualified as Inps
+import Lang.Crucible.Debug.Statement (Statement)
+import Lang.Crucible.Simulator.ExecutionTree qualified as C
+
+-- | A class for Crucible personality types @p@ which contain a
+-- 'RecordState'. This execution feature is polymorphic over
+-- 'RecordState' so that downstream users can supply their own
+-- personality types that extend 'RecordAndReplay' further.
+class HasContext p cExt sym ext t | p -> cExt sym ext t where
```

Comment:
Instead of `RecordState`, should these Haddocks mention `Context`?

A similar question about the mention of `RecordAndReplay` here, since I think that is a copy-paste leftover.

Reply:
Whoops, thanks for the catch!